src/main.js

```diff
@@ -44,16 +45,15 @@ let load = base.loader({
         referencePrefix:  'taskcluster-treeherder/v1/exchanges.json',
         publish:          cfg.app.publishMetaData,
         aws:              cfg.aws,
-        drain:            drain,
-        component:        cfg.app.statsComponent,
+        monitor: monitor.prefix('publisher'),
         process,
```

Comment:
I don't think exchanges needs `process` anymore. It was used for the old stats stuff. It should be yelling about this, but maybe I missed it.